enatega-multivendor-app/.env

```
@@ -0,0 +1,9 @@
+# Customer Google OAuth
+EXPO_PUBLIC_GOOGLE_WEB_CLIENT_ID=505610605704-ne45g8fl1cclnbh0drl7oirjb8vjai3b.apps.googleusercontent.com
+EXPO_PUBLIC_GOOGLE_ANDROID_CLIENT_ID=505610605704-8u95anr6r2rvd2gif7cdohqc00nkau9n.apps.googleusercontent.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_GOOGLE_ANDROID_CLIENT_ID key should go before the EXPO_PUBLIC_GOOGLE_WEB_CLIENT_ID key

(UnorderedKey)